_posts/blog/2017-06-29-accumulo-documentation-improvements.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Documentation Improvements for 2.0"
+author: Mike Walch
+---
+
+Since Accumulo 1.5, the [Accumulo user manual][manual] source has resided in the [Accumulo repository][accumulo-repo] as asciidoc. For every release or update to the manual,
+an HTML document is produced from the asciidoc, committed to the [Accumulo website repository][website-repo], and published to the Accumulo website. This process will remain
+for 1.x releases.
+
+Starting with the upcoming 2.0 release, the source for the Accumulo user manual was converted to markdown and moved to Accumulo website repository. The
```

Comment:
Could be :

    For 2.0, the source for the user manual was converted to markdown and moved to the [website repository][webrepo]

I think the `Accumulo` mentions can be dropped. Maybe the first time the user manual is mentioned call it the `Accumulo user manual`.  After that in doc, just call it the `user manual`.  Same for website repo.